File: common/src/main/java/app/dropbit/commons/currency/CryptoCurrency.java
package app.dropbit.commons.currency;


import android.content.Context;
import android.graphics.drawable.Drawable;


public interface CryptoCurrency extends Currency {
    String NO_SYMBOL_FORMAT = "#,##0.########";

    Drawable getSymbolDrawable(Context context);

    FiatCurrency toFiat(FiatCurrency conversionFiat);
}

File: common/src/main/java/app/dropbit/commons/currency/CryptoCurrency.kt
package app.dropbit.commons.currency

import android.content.Context
import android.graphics.drawable.Drawable

interface CryptoCurrency : Currency {
    fun getSymbolDrawable(context: Context): Drawable?
    fun toFiat(conversionFiat: FiatCurrency): FiatCurrency

    companion object {
        const val NO_SYMBOL_FORMAT = "#,##0.########"
    }
}